Kickstarter-iOS/Features/DashboardDeprecationBanner/DashboardDeprecationView.swift

```diff
@@ -0,0 +1,50 @@
+import Library
+import SwiftUI
+
+struct DashboardDeprecationView: View {
+  private let contentPadding = 12.0
+  private let imageSizeMultiplier = 1.5
```

Comment:
Try to use constants for hardcoded values at the top of the file, makes changing the same thing in multiple places easier and is more readable.